wolox-books/src/app/login/login.component.html

```diff
@@ -0,0 +1,16 @@
+<div class="from-container">
+  <img class="logo-wolox" alt="logo-wolox" src="assets/LogoWolox-Original.png">
+  <form [formGroup]='registerForm' (ngSubmit)='addPost()'>
+    <div class="section-input" *ngFor="let item of inputData">
+      <label class="text-label">{{ item.text }}</label>
+      <input class="decoration-input" type="{{ item.type }}" formControlName="{{ item.formControlName }}"/>
```

Comment:
Don't use string interpolation in html attributes, you should put the property in square brackets ([]), eg: [type]="variableName", string interpolation is only used when you are putting dynamic text in the application :)